<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmForm</class>
 <widget class="QWidget" name="FilmForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 53, 53);

background-color: qlineargradient(spread:pad, x1:0.249692, y1:0.215, x2:1, y2:1, stop:0.259615 rgba(63, 63, 63, 255), stop:1 rgba(120, 120, 120, 255));
border: 2px solid ;
border-bottom:2px solid yellow;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QLabel" name="FilmImage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(212, 212, 0);
border-color: rgb(255, 255, 0);
border-radius:10px;
padding-left:1px;
border:5px solid yellow;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="FilmTitle">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;
border:0px solid;
color:yellow;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="FilmDescription">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
border:0px solid;
color:yellow;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Like">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(212, 212, 0);
border-color: rgb(255, 255, 0);
border-radius:10px;
margin:10px;
padding-left:15px;
padding-right:15px;</string>
         </property>
         <property name="text">
          <string>Like</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Dislike">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(212, 212, 0);
border-color: rgb(255, 255, 0);
border-radius:10px;
margin:10px;
padding-left:15px;
padding-right:15px;</string>
         </property>
         <property name="text">
          <string>Dislike</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
